app/assets/stylesheets/new_board/_story.scss

```diff
@@ -0,0 +1,135 @@
+.Story {
+  display: flex;
+  justify-content: space-between;
+  padding: 7px 8px;
+  background-repeat: no-repeat;
+  background-position: 4px 4px;
+  background-color: $story-background-color;
```

Comment:
We can use just `background: $story-background-color 4px 4px no-repeat`. But I think that background-repeat and the background-position are not necessary since we are using a color. So, we can just put `background: $story-background-color`.

Reply:
Nice, fits better!